### PR TITLE
fix adobe.com #176525

### PR DESCRIPTION
**Pull request by slavaleleka**
https://github.com/AdguardTeam/AdguardFilters/issues/176525

### CODE REVIEW

File: AnnoyancesFilter/Other/sections/self-promo.txt [hunk 1]
```diff
@@ -137,6 +137,8 @@ www.kinopoisk.ru##div[class^="trailer-promo"]
 ! END: Yandex
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/176525
+adobe.com##.promobarModal
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/Other/sections/self-promo.txt:141]: to popups maybe?

Comment (slavaleleka) [reply AnnoyancesFilter/Other/sections/self-promo.txt:141]: even if it's actually a self-promo?

Comment (Sergey-Lyapin) [reply AnnoyancesFilter/Other/sections/self-promo.txt:141]: as well as a popup

Comment (zloyden) [reply AnnoyancesFilter/Other/sections/self-promo.txt:141]: I think it's ok to move it to popups.

Comment (slavaleleka) [reply AnnoyancesFilter/Other/sections/self-promo.txt:141]: so 'popup' prevails over 'self-promo' in this case and the rule should be moved to 
`AnnoyancesFilter/Popups/sections/popups_specific.txt` ?

Comment (Alex-302) [reply AnnoyancesFilter/Other/sections/self-promo.txt:141]: Yes.

**Review by zloyden — APPROVED** [review]

**Review by Sergey-Lyapin — APPROVED** [review]

**Pull request merged**